judgels-frontends/raphael/src/components/ProblemWorksheetCard/Bundle/ProblemStatementCard/ItemEssayCard/ItemEssayCard.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { ReactWrapper, mount } from 'enzyme';
+import { ItemEssayCard, ItemEssayCardProps } from './ItemEssayCard';
+import { ItemType, ItemEssayConfig } from 'modules/api/sandalphon/problemBundle';
+import * as React from 'react';
+
+describe('ItemEssayCard', () => {
+  let wrapper: ReactWrapper<ItemEssayCard>;
+  const itemConfig: ItemEssayConfig = {
+    statement: 'statement',
+    score: 100,
+  };
+  const props: ItemEssayCardProps = {
+    jid: 'jid',
+    type: ItemType.Essay,
+    meta: 'meta',
+    config: itemConfig,
+    disabled: false,
+    onSubmit: jest.fn().mockReturnValue(undefined),
+    itemNumber: 1,
+  };
+
+  beforeEach(() => {
+    wrapper = mount(<ItemEssayCard {...props} />);
+  });
+
+  it('item statement', () => {
```

Comment:
We should not test implementation details like this. These tests are dependent on the classs names (`__item-xxx`).

We can combine both the following 2 tests, by just testing that the `.text()` is equal to `1. statement`.